api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -77,4 +79,26 @@ public boolean isUsernameTaken(String username) {
       throw new RuntimeException(e);
     }
   }
+
+  public void createUser(String givenName, String familyName, String username, String password) {
+    String gSuiteDomain = configProvider.get().googleDirectoryService.gSuiteDomain;
+    User user = new User()
+      .setPrimaryEmail(username+"@"+gSuiteDomain)
+      .setPassword(password)
+      .setName(new UserName().setGivenName(givenName).setFamilyName(familyName));
```

Comment:
I think we're going to want to set a contact email here, too, in the emails field.

Reply:
I don't think we should add it until it has been verified. Whether or not we require it as part of the registration process, I don't think it should be part of this particular call.

Reply:
How do you expect to verify the e-mail without first writing it to the database for the user?

(I think the right thing is to store the e-mail initially and have a flag indicating when it's been verified.)

Reply:
Note: I'm not necessarily saying you have to store it in their Google account metadata. So you could leave it out of this PR. But I do think we're going to want to capture e-mail address on initial sign up, if not gate on verification of it.

Reply:
Also, FWIW: I was able to set a recovery email (but not an alternate email) for my pmi-ops.org Google account.  On my consumer account, I can add an alternate email, and it's in a pending state awaiting verification.

I don't know what all this means for our accounts. My guess is we'll have to do our own e-mail verification logic if we want it.

Reply:
Good points on verification. I'd like to punt on it for this PR.